server/routes/taskList.js

```diff
@@ -35,14 +38,19 @@ module.exports = ({ prisonerService, licenceService, audit }) => router => {
       const licenceStatus = getLicenceStatus(licence)
       const allowedTransition = getAllowedTransition(licenceStatus, req.user.role)
       const { statusLabel } = getStatusLabel(licenceStatus, req.user.role)
+      let taskListModel
+      let errors
 
-      const taskListModel = getTaskListModel(
-        req.user.role,
-        postRelease,
-        licenceStatus,
-        licence || {},
-        allowedTransition
-      )
+      if (licenceStatus.stage === 'ELIGIBILITY' && req.user.role === 'CA') {
+        const errorCodesList = await caService.getReasonForNotContinuing(bookingId, res.locals.token)
+        errors = errorCodesList ? errorCodesList.map(error => taskListErrors[error]) : null
```

Comment:
think rather than default to null, this could default to an empty array - then you won't have to null check in the templates.

Reply:
routes/taskList.js :- line 44, replaced ternary with if()
routes/viewModels/taskListModels.js (errors) changed to (errors && errors.length > 0)